packages/experiments/src/components/Chiclet/ChicletCard.types.ts

```diff
@@ -105,6 +109,12 @@ export interface IChicletCardStyleProps {
    */
   theme: ITheme;
 
+  /**
+   * Accept image props.
```

Comment:
Could we add individual comments for both `imageProvided` and `imageAltProvided`? Maybe it should be something more like "Whether image is provided" and "Whether imageAlt is provided"